src/unit_tests/os_auth/test_auth_validate.c

```diff
@@ -100,6 +102,17 @@ static int setup_group(void **state) {
 static int teardown_group(void **state) {
     OS_FreeKeys(&keys);
 
+    struct keynode *cur = NULL;
+    struct keynode *next = NULL;
+
+    for (cur = queue_remove; cur; cur = next) {
+        next = cur->next;
+        free(cur->id);
```

Comment:
I think using os_free here it will make the tests easier to sustain.

Reply:
You are right, fixed!